### PR TITLE
Feature/scripts for UI

### PR DESCRIPTION
**Pull request by AndriiStasiuk**

### CODE REVIEW

**Review by MaksymMartyniak — APPROVED** [review]

**Review by Liliia111 — APPROVED** [review]

**Review by filipenm — APPROVED** [review]

File: SummerProject/static/src/components/CheckEmail/index.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import './style.css';
 class CheckEmail extends React.Component {
 
     render() {
+        console.log(this.props);
```
Comment (UlianaShevtsiv) [inline SummerProject/static/src/components/CheckEmail/index.js]: `console.log` ? 😮

File: SummerProject/user/validator.py [hunk 2]
```diff
@@ -56,7 +56,9 @@ def is_valid_password_for_reset(data):
         if k not in params:
             print(k)
```
Comment (UlianaShevtsiv) [inline SummerProject/user/validator.py]: `print` 😞

**Review by UlianaShevtsiv — APPROVED** [review]
+1. Great job!

File: SummerProject/user/validator.py (hunk 2 above)
Comment (AndriiStasiuk) [reply SummerProject/user/validator.py]: It's not my code)

**Pull request merged**